README.md

```diff
@@ -1,9 +1,13 @@
 Before compiling do these:
 
+### Perlmutter
+```sh
 module load PrgEnv-cray cudatoolkit craype-accel-nvidia80
 export CRAY_ACCEL_TARGET=nvidia80
-
-When running do these:
-
-module load cudatoolkit
 export MPICH_GPU_SUPPORT_ENABLED=1
+```
+### Frontier
+```sh
+module load PrgEnv-cray amd-mixed craype-accel-amd-gfx90a
```

Comment:
do we want to load a specific version of ROCM here? the default is 5.3.0 which is a bit low maybe